app/views/user_mailer/in_person_verified.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= image_tag(
+      'email/user-signup-ial2.png',
+      width: 140,
+      height: 177,
+      alt: '',
+      class: 'float-center padding-bottom-4',
+    ) %>
+<h4><%= message.subject %></h4>
+<p>
+<div class="s16">
+  <%= t('user_mailer.in_person_verified.greeting') %><br>
+  <%= t(
+        'user_mailer.in_person_verified.intro',
+        location: @presenter.location_name,
+        date: @presenter.formatted_verified_date,
+      ) %><br /><br />
+  <%= t('user_mailer.in_person_verified.next_sign_in', app_name: APP_NAME) %>
+</div>
+</p>
+
+<div class="mt-40">
+<table class="button expanded large radius">
```

Comment:
Could we collapse the margin class into the table?

```suggestion
<table class="button expanded large radius margin-top-5">
```

Reply:
I tried this change, but the margin was being overridden by an inline style. I've left it as-is for now.